src/Swashbuckle.AspNetCore.SwaggerGen/XmlComments/XmlCommentsTextHelper.cs

```diff
@@ -125,6 +129,7 @@ private static string DecodeXml(this string text)
         private const string ParaTagPattern = @"<para>(?<display>.+?)</para>";
         private const string HrefPattern = @"<see href=\""(.*)\"">(.*)<\/see>";
         private const string BrPattern = @"(<br ?\/?>)"; // handles <br>, <br/>, <br />
+        private const string LineBreaksPattern = @"\r\n?|\n";
```

Comment:
Shame we can't use `string.ReplaceLineEndings()`.

This could be simplified to this? `\r?\n`. I'm not sure whether anyone still actively uses `\r` for a new line on modern macOS?

Reply:
> I'm not sure whether anyone still actively uses \r for a new line on modern macOS?

I think you're right, simplified